resources/main.css

```diff
@@ -1,3 +1,13 @@
+a {
+    color: white;
+    font-family: Roboto,sans-serif;
+}
+
+button {
```

Comment:
As this is the only button that has this special styling I'd say you should place the CSS inside of the HTML or use identifiers to be able to place the styling in `options.css` (which does not exist right now)